exporter/azuremonitorexporter/traceexporter.go

```diff
@@ -71,6 +71,12 @@ const (
 	spanAttributeKeyDbInstance  = "db.instance"
 	spanAttributeKeyDbStatement = "db.statement"
 	spanAttributeKeyDbUser      = "db.user"
+
+	// Resource keys
+	// https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-resource-semantic-conventions.md
+	resourceAttributeKeyServiceName       = "service.name"
+	resourceAttributeKeyServiceNamespace  = "service.namespace"
+	resourceAttributeKeyServiceInstanceID = "service.instance.id"
```

Comment:
These are already defined in https://github.com/open-telemetry/opentelemetry-collector/blob/master/translator/conventions/opentelemetry.go
Any reason for redefining and not importing?

Reply:
No, I'll definitely import.